cobc/flag.def

```diff
@@ -141,6 +141,10 @@ CB_FLAG (cb_flag_traceall, 1, "traceall",
 CB_FLAG (cb_flag_syntax_only, 1, "syntax-only",
 	_("  -fsyntax-only         syntax error checking only; don't emit any output"))
 
+CB_FLAG (cb_flag_preproc_area_a, 1, "preproc-area-a",
```

Comment:
Do you mean we should always output the `#` whenever not in free format?

Reply:
No, depending on the -fformat (with / without area a enforcement) this should be inserted and later on checked.
Note: if we need another -fformat option to distinguish cobol85 and fixed - then it would be good to add that soon, even if they currently are identical (documenting the first as "with area a enforcement [currently not done]", or similar).